pyproject.toml

```diff
@@ -15,7 +15,7 @@ python = "^3.6"
 importlib-metadata = { version = "*", python = "<3.8" }
 
 # Base requirements
-neptune-client = ">=0.16.7"
+neptune-client = ">=0.16.16"
```

Comment:
Probably `0.16.17` will be better, as there were some fixes to the `stringify_unsupported` function made

Reply:
Right! Thank you :)